docs/user-manual/index.md

```diff
@@ -782,7 +782,7 @@ specific behavior in various situations.
 | `-v`<br />`--version` | Show program's version number and exit.  |
 | `-t`<br />`--test-mode` | Run API action calls in test mode (does not execute changes). Logs what would have been executed.  |
 | `-c` _filename_<br />`--config-filename` _filename_ | The complete path to the main configuration file, absolute or relative to the working folder. Default filename is "user-sync-config.yml" |
-| `--users` `all`<br />`--users` `file` _input_path_<br />`--users` `group` _grp1,grp2_ | Specify the users to be selected for sync. The default is `all` meaning all users found in the directory. Specifying `file` means to take input user specifications from the CSV file named by the argument.  Specifying `group` interprets the argument as a comma-separated list of groups in the enterprise directory, and only users in those groups are selected. |
+| `--users` `all`<br />`--users` `file` _input_path_<br />`--users` `mapped`<br />`--users` `group` _grp1,grp2_ | Specify the users to be selected for sync. The default is `all` meaning all users found in the directory. Specifying `file` means to take input user specifications from the CSV file named by the argument. Specifying `mapped` is the same as specifying `group` with all groups listed in the configuration file. Specifying `group` interprets the argument as a comma-separated list of groups in the enterprise directory, and only users in those groups are selected. |
```

Comment:
move `mapped` *after* `group` (both on the left side and the right side), because the definition of mapped depends on the definition of group.  I also suggest that rather than the phrase "listed in the configuration file" you say "listed in the group mapping in the configuration file".  It means the same thing, but makes it clear where to find them.